### PR TITLE
fix: run generate_countries to fix GB being flagged as eu_member

### PR DESCRIPTION
**Pull request by dotbear**

### CODE REVIEW

**Review by jgwmaxwell — APPROVED** [review]

**Review by nikcorg — APPROVED** [review]

File: tasks/generate_data.ex [hunk 1]
```diff
@@ -71,26 +78,28 @@ defmodule Mix.Tasks.Countriex.GenerateData do
...
-      min_latitude:   geo_data.min_latitude |> to_float,
-      min_longitude:  geo_data.min_longitude |> to_float,
+      latitude: geo_data.latitude |> to_float,
+      latitude_dec: geo_data.latitude_dec |> to_float,
+      longitude: geo_data.longitude |> to_float,
+      longitude_dec: geo_data.longitude_dec |> to_float,
+      max_latitude: geo_data.max_latitude |> to_float,
+      max_longitude: geo_data.max_longitude |> to_float,
+      min_latitude: geo_data.min_latitude |> to_float,
+      min_longitude: geo_data.min_longitude |> to_float
```
Comment (nikcorg) [inline tasks/generate_data.ex:91]: for consistency with other edits, maybe also

```suggestion
      latitude: geo_data.latitude |> to_float(),
      latitude_dec: geo_data.latitude_dec |> to_float(),
      longitude: geo_data.longitude |> to_float(),
      longitude_dec: geo_data.longitude_dec |> to_float(),
      max_latitude: geo_data.max_latitude |> to_float(),
      max_longitude: geo_data.max_longitude |> to_float(),
      min_latitude: geo_data.min_latitude |> to_float(),
      min_longitude: geo_data.min_longitude |> to_float()
```

File: tasks/generate_data.ex [hunk 2]
```diff
@@ -71,26 +78,28 @@ defmodule Mix.Tasks.Countriex.GenerateData do
...
-      max_latitude:   geo_data.max_latitude |> to_float,
-      max_longitude:  geo_data.max_longitude |> to_float,
-      min_latitude:   geo_data.min_latitude |> to_float,
-      min_longitude:  geo_data.min_longitude |> to_float,
+      latitude: geo_data.latitude |> to_float,
+      longitude: geo_data.longitude |> to_float,
+      max_latitude: geo_data |> Map.get(:max_latitude) |> to_float,
+      max_longitude: geo_data |> Map.get(:max_longitude) |> to_float,
+      min_latitude: geo_data |> Map.get(:min_latitude) |> to_float,
+      min_longitude: geo_data |> Map.get(:min_longitude) |> to_float
```
Comment (nikcorg) [inline tasks/generate_data.ex:102]: ...and...

```suggestion
      latitude: geo_data.latitude |> to_float(),
      longitude: geo_data.longitude |> to_float(),
      max_latitude: geo_data |> Map.get(:max_latitude) |> to_float(),
      max_longitude: geo_data |> Map.get(:max_longitude) |> to_float(),
      min_latitude: geo_data |> Map.get(:min_latitude) |> to_float(),
      min_longitude: geo_data |> Map.get(:min_longitude) |> to_float()
```

File: tasks/generate_data.ex [hunk 3]
```diff
@@ -7,49 +7,55 @@ defmodule Mix.Tasks.Countriex.GenerateData do
...
+
+        404 ->
+          []
       end
     end)
   end
 
-  defp assign_state_code(data) when is_map(data) do
+  defp assign_state_code(data) do
+    {:ok, data} = data
```
Comment (nikcorg) [inline tasks/generate_data.ex:53]: why not?

```suggestion
  defp assign_state_code({:ok, data}) do
```

**Comment by nikcorg** [comment]
Added a few comments, but they're only stylistic refactorings.

**Pull request merged**